controllers/storageclaim_controller.go

```diff
@@ -274,13 +274,13 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 
 		// storageClaimStorageType is the storage type of the StorageClaim
 		var storageClaimStorageType providerclient.StorageType
-		switch r.storageClaim.Spec.Type {
+		switch strings.ToLower(r.storageClaim.Spec.Type) {
```

Comment:
Please calculate the lowered case value only once and use it in both places.

Reply:
done.